CMakeLists.txt

```diff
@@ -619,19 +614,6 @@ if(CUDAQ_BUILD_TESTS)
   umbrella_lit_testsuite_end(check-all)
 endif()
 
-if (CUDAQ_BUILD_RELOCATABLE_PACKAGE AND NOT CUDAQ_DISABLE_CPP_FRONTEND) 
```

Comment:
How will this work if one has more than 1 clang++ installed on the system?

Reply:
The CMAKE_INSTALL_PREFIX set for the LLVM build determines where LLVM and the clang toolchain is installed. For the build scripts, this is configurable via the LLVM_INSTALL_PREFIX variable. 
Aside from the fact that the installer still required root privileges, this will not interfere with any other compiler on the system, since there is no need to have that clang version or any of its libraries be discovered via system paths / the usual environment variables. The installation is still not fully relocatable (it never was), but the issue there is not llvm/clang (which we can relocate into the cudaq install folder) but rather other dependencies (e.g. cuQuantum).

Reply:
I think I made this comment no knowing that this RELOCATABLE_PACKAGE stuff was all being tossed in the bin. Now that I know that: no worries.